### ISSUE TITLE
No entity manager defined for class  Sympal\TorneioBundle\Entity\Tournaments

### ISSUE THREAD

**Issue by dsteplight** [sonata-project/SonataAdminBundle#3204]
When I connect FOSUserBundle with SonataAdmin bundle I'm seeing this error when I click on the link_add or link_list links for my content. These links were working properly before I added the FOSUserBundle.  Is there a config change that I'm missing?  I looked over the documentation over and over , but where should the entity manager mapping be set? I tried in the app/confg/config.yml file.  Here is a link to the gist https://gist.github.com/dsteplight/104b58c39be61695d0e1 .

**Comment by tim96** [comment]
@dsteplight I think you need to try uncomment line in config.yml, but I'm not sure: 
```
auto_mapping: true
```

**Comment by dsteplight** [comment]
@tim96  That causes an Exception to be thrown "InvalidConfigurationException in ArrayNode.php line 309:
Unrecognized option "auto_mapping" under "doctrine.orm"".

**Comment by tim96** [comment]
@dsteplight I have such configuration and it working without exception. What version bundles do you use? : 
```
doctrine:
    orm:
        entity_managers:
            default:
                auto_mapping: true
```

**Comment by dsteplight** [comment]
@tim96 
    "require": {
        "php": ">=5.3.9",
        "symfony/symfony": "2.7.*",
        "doctrine/orm": "~2.2,>=2.2.3,<2.5",
        "doctrine/dbal": "<2.5",
        "doctrine/doctrine-bundle": "~1.4",
        "symfony/assetic-bundle": "~2.3",
        "symfony/swiftmailer-bundle": "~2.3",
        "symfony/monolog-bundle": "~2.4",
        "sensio/distribution-bundle": "~4.0",
        "sensio/framework-extra-bundle": "~3.0,>=3.0.2",
        "incenteev/composer-parameter-handler": "~2.0",
        "sonata-project/admin-bundle": "^2.3",
        "sonata-project/doctrine-orm-admin-bundle": "^2.3",
        "sonata-project/easy-extends-bundle": "^2.1",
        "sonata-project/user-bundle": "^2.2"
    },
    "require-dev": {
        "sensio/generator-bundle": "~2.3"
    },

**Comment by dsteplight** [comment]
@tim96  I ended up starting from scratch and paying closer attention to the documentation but you were on the right track. I needed to updated my config.yml file to use auto_loading. The documentation shows how to do it without auto_loading which I followed in my config but in parentheses it say OR just turn "auto_mapping" which made life easier.     

The following snippet from my doctrine section in my config file got things working for me.
orm:
        auto_generate_proxy_classes: "%kernel.debug%"
        naming_strategy: doctrine.orm.naming_strategy.underscore
        auto_mapping: true
